src/app/public/modules/popover/popover.component.ts

```diff
@@ -106,6 +106,8 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
 
   public classNames: string[] = [];
 
+  public containOverflow = false;
```

Comment:
Would `containOverflow` be a good candidate for `@Input`? I’m not sure it's good practice to have public properties on components that are designed to be used by other components. Ideally, a component’s API would be locked down to `Input` and `Output` only.

The name of the property is better than it was before, but I would argue that "fullscreen" mode _also_ contains the overflow of the contents, so setting `containOverflow` still doesn't communicate the expected behavior. Since we're preventing the popover from going fullscreen, what if we named it `allowFullscreen`?

```
@Input()
public allowFullscreen = true;
```

Reply:
Yeah I like that. Let's do it.

Reply:
Done.